File: TwoSum.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    std::vector<int> twoSum(std::vector<int>& nums, int target) {
        for(auto i = 0; i < nums.size(); i++){
            for(auto j = i + 1; j < nums.size(); j++){
                if(nums[i] + nums[j] == target){
                    std::vector<int> Result {i, j};
                    return Result;
                }
            }
        }
        return std::vector<int>{0, 0};
    }
};

int main() {
    std::cout << "Hello, World!" << std::endl;
    return 0;
}

File: easy/two-sum/two-sum.go
package two_sum

func twoSum(nums []int, target int) []int {
	var index []int
	for i := 0; i < len(nums); i++ {
		for j := i + 1; j < len(nums); j++ {
			if (nums[i] + nums[j]) == target {
				index = append(index, i, j)
				return index
			}
		}
	}

	return []int{0, 0}
}
